setup.py

```diff
@@ -1,6 +1,7 @@
 import os
 import warnings
 from packaging import version
+import versioneer
```

Comment:
Is there any overlap in the functionality of these two packages? Can we achieve the same by only using one of the two?

Reply:
I don't think so. Especially, if we use `versioneer.py`. I quickly skimmed through the file and do not have the impression that the functionalty of `packaging.version` (concatenation/comparison of versions) is provided here.